ping-protect/src/main/java/org/forgerock/android/auth/PingOneProtectEvaluationCallback.kt

```diff
@@ -8,21 +8,19 @@ package org.forgerock.android.auth
 
 import android.content.Context
 import androidx.annotation.Keep
-import org.forgerock.android.auth.callback.AbstractCallback
+import org.forgerock.android.auth.callback.AbstractProtectCallback
```

Comment:
Remove unnecessary import

Reply:
you mean AbstractProtectCallback? we need this import